cmd/down.go

```diff
@@ -27,11 +27,6 @@ func down(cmd *cobra.Command, args []string) {
 	//read the harbor compose file
 	var harborCompose = DeserializeHarborCompose(HarborComposeFile)
 
-	_, _, err := Login()
```

Comment:
If can be but it adds more baggage to the code to have to pass that around everywhere.  It's much more isolated the way it's coded now.